code/lib/blocks/src/examples/Button.stories.tsx

```diff
@@ -116,10 +117,18 @@ export const ErrorStory: Story = {
   render: () => {
     const err = new Error('Rendering problem');
     // force stack for consistency in capture
-    err.stack = err.stack
-      .replace(/\d+:\d+(:\d+)?/g, `000:0001`)
-      .replace(/v=[^:]+/g, 'v=00000000')
-      .replace(/[^/]+\.js/g, 'file.js');
+    err.stack = `
```

Comment:
maybe `dedent` it?

Reply:
The error stack is automatically dedented natively, there was no need to add dedent to it!